P4Plugin/Source/P4OSX.mm

```diff
@@ -0,0 +1,20 @@
+#include "P4Task.h"
+#import <AppKit/AppKit.h>
+
+bool P4Task::ShowOKCancelDialogBox(const std::string& windowTitle, const std::string& message)
+{
+    NSString *nsWindowTitle = [NSString stringWithUTF8String:windowTitle.c_str()];
+    NSString *nsMessage = [NSString stringWithUTF8String:message.c_str()];
+
+    CFOptionFlags cfRes;
+    CFUserNotificationDisplayAlert(0, kCFUserNotificationNoteAlertLevel,
+            NULL, NULL, NULL,
+            (__bridge CFStringRef)nsWindowTitle,
+            (__bridge CFStringRef)nsMessage,
+            CFSTR("Cancel"),
+            CFSTR("Ok"),
```

Comment:
On Linux/Windows the label is all uppercase "OK", maybe should be like that on Mac too

Reply:
Fixed in 11da728